starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort();
+    this.length += 1;
+  }
+  get(pos) {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.item[this.length - 1];
+    }
+  }
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.max(this.items);
```

Comment:
to return the max, you can use the spread operator `  return Math.max(...this.items); `  or `this.items[this.length-1]` 

Returning the length -1 works because you've already sorted the array